src/test/resources/results/scriptCommand/testScriptCommandToGenerateScripts/Employee/Makefile

```diff
@@ -37,7 +37,7 @@ first-run:
 	$(MAKE) copy-into
 	$(MAKE) create-snowpipe
 
-clean:
+drop:
```

Comment:
Looking at this diff I feel that the  master in your local is not updated. 
https://github.com/phdata/streamliner/commit/a20fa9e19665729b21a4ac2a05cb4474c1ada4f6#diff-4798b65a56e4bcec6a83feaeb58286adb9abcb833a1d7b0c91f99b65559e8975

I think this will create conflict while merge.